lib/money/currency.rb

```diff
@@ -201,6 +201,24 @@ def unregister(curr)
         existed ? true : false
       end
 
+      # Returns if a code currency is ISO.
+      #
+      # @param [Symbol] code currency
+      #
+      # @return [Boolean]
+      #
+      # @example
+      #   Money::Currency.is_iso?(:eur)
+      #
+      def is_iso?(id)
```

Comment:
nit: I think `iso?` would be a bit more idiomatic ruby — `?` already suggests that it returns a boolean result, no need for `is_`